package/CHANGELOG

```diff
@@ -33,6 +33,7 @@ Fixes
  * Fixes the benchmark `SimpleRmsBench` in `benchmarks/analysis/rms.py`
    by changing the way weights for RMSD are calculated, instead of
    directly passing them. (Issue #3520, PR #5006)
+ * Fix incorrect `self.atom` assignment in SingleFrameReaderBase (Issue #5052, PR #5055)
```

Comment:
Sorry to be pedantic about this, but could you put this at the top? It's very useful on release to get the chronological order right.